framework/kibocommerce/api/endpoints/wishlist/index.ts

```diff
@@ -0,0 +1,24 @@
+import { GetAPISchema, createEndpoint } from '@commerce/api'
+import wishlistEndpoint from '@commerce/api/endpoints/wishlist'
+// import type { WishlistSchema } from '../../../types/wishlist'
```

Comment:
we probably should remove this commented code

Reply:
removed